css/style.css

```diff
@@ -1,3 +1,16 @@
 h1 {
-  color: purple;
+  color: black;
+}
+.gameboard{
+  display: grid;
+  grid-template: 1fr 1fr 1fr / 1fr 1fr 1fr;
+  flex-flow: row wrap;
+  width: 630px;
+  height:630px
+}
+.square{
+  width:210px;
+  height:210px;
+  margin: 0px;
+  border: 5px solid black;
 }
```

Comment:
Just another note on writing cleaner code:

You will see all of the instructors in this course have a space between each set of CSS rules (so you would have one blank line between lines 3 and 4 and another before your line that says ".square{"

Also, we place a space before the opening curly brace { 

And one final note that a couple of your attributes and values are missing spaces after the colon's.

And you're also missing a semicolon on line 9.  None of these things will break your code but will make your code look a lot more presentable/easier to read and will show potential employers that view your github that you pay attention to detail.